packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -761,4 +874,6 @@ export default connect(mapStateToProps, {
   setActiveTool: setActiveToolAction,
   updateDashboardData: updateDashboardDataAction,
   updateWorkspaceData: updateWorkspaceDataAction,
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
As mentioned in #ddl-team-web, you shouldn't need these. Instead declare AppMainContainer to have the RouteComponentProps:
```
export class AppMainContainer extends Component<
  AppMainContainerProps & RouteComponentProps,
  AppMainContainerState
> {
```

Then you'll also need to cast the props from redux correctly, e.g.:
```
const mapStateToProps = (state: RootState) => ({
  activeTool: getActiveTool(state),
  dashboardData: getDashboardData(state, DEFAULT_DASHBOARD_ID) as AppDashboadData,
  layoutStorage: getLayoutStorage(state) as LayoutStorage,
  plugins: getPlugins(state),
  session: getDashboardSessionWrapper(state, DEFAULT_DASHBOARD_ID).session,
  sessionConfig: getDashboardSessionWrapper(state, DEFAULT_DASHBOARD_ID).config,
  user: getUser(state) as User,
  workspace: getWorkspace(state),
});

```